### PR TITLE
Added time take to run a process.

### LINKED ISSUE #4: Show time taken for various actions
I would like to add a feature where the time taken to generate the stdin and stdout files are being outputted. 

For example here is the current output:
```
Generating stdin for subtask '1', testcase '1'...
Saved 'testcases\1-1.in'.
Generating stdin for subtask '1', testcase '2'...
Saved 'testcases\1-2.in'.
Generating stdin for subtask '1', testcase '3'...
Saved 'testcases\1-3.in'.
```

We can add time taken such that its now something like:
```
Generating stdin for subtask '1', testcase '1'...
Saved 'testcases\1-1.in'. Took 0.XXXXs.
Generating stdin for subtask '1', testcase '2'...
Saved 'testcases\1-2.in'. Took 0.XXXXs.
Generating stdin for subtask '1', testcase '3'...
Saved 'testcases\1-3.in'. Took 0.XXXXs.
```

### PR DESCRIPTION
**Pull request by abhijeetgupto**
As described in one of your issues , added time take to complete a process .

### CODE REVIEW

**Comment by benwoo1110** [comment]
Looks good! Can you add the same for `generate_stdout` as well?

**Comment by abhijeetgupto** [comment]
Sure , I'll do that .

**Comment by abhijeetgupto** [comment]
Can you please check again . I did those changes .

File: testcase_maker/generator.py [hunk 1]
```diff
@@ -87,7 +90,8 @@ def generate_stdin(self, override: bool = False):
                 with open(stdin_file, "w", newline="\n") as input_buffer:
                     input_buffer.write(stdin)
                     input_buffer.write("\n")
-                print(f"Saved '{stdin_file}'.")
+                end = timer()
+                print(f"Saved '{stdin_file}'. Took {end-start} seconds .")
```
Comment (benwoo1110) [inline testcase_maker/generator.py]:
```suggestion
                print(f"Saved '{stdin_file}'. Took {end-start} seconds.")
```
remove extra space here.

**Review by benwoo1110 — COMMENTED** [review]
Just a minor nitpick before I can merge ;)

**Comment by abhijeetgupto** [comment]
Done :)

**Review by benwoo1110 — APPROVED** [review]
Tested working. Thank you for contributing, appreciate it!

**Comment by abhijeetgupto** [comment]
Can you please add HacktoberFest Accepted label before closing !

**Comment by benwoo1110** [comment]
merging will be considered accepted by HacktoberFest. But sure I can add the label. 

(also HacktoberFest servers are quite slow rn so it may take 15-30min for them to update that the pr is merged)

happy coding! 😄

**Comment by abhijeetgupto** [comment]
Thanksss :)

**Pull request merged**